modules/ROOT/pages/patterns/shortest-paths.adoc

```diff
@@ -538,3 +538,84 @@ RETURN [stop IN nodes(p) | stop.name] AS r1,
 
 2+d|Rows: 1
 |===
+
+
+[[planning-StatefulShortestPath-operator]]
+== Planning the StatefulShortestPath-operator
+See xref:planning-and-tuning/operators/operators-detail.adoc#query-plan-stateful-shortest-path-into[the operator-detail page] for a description of the `StatefulShortestPath(All)` and `StatefulShortestPath(Into)` operators.
```

Comment:
I would still add one sentence about the algorithms here and then point to the link for further details:

```
The `StatefulShortestPath(All)` uses a unidirectional breadth first search algorithm, while the `StatefulShortestPath(Into)` uses a bidirectional breadth first search algorithm.
```